packages/utils/util.ts

```diff
@@ -156,3 +156,18 @@ export function getRandomInt(max: number) {
 }
 
 export { isVNode } from 'vue'
+
+// TODO: use native Array.find, Array.findIndex when IE support is dropped
```

Comment:
Just use Array.find, please

Reply:
Ok!
Optimization will be carried out later, including type checking